content/applications/sales/subscriptions/ecommerce.rst

```diff
@@ -19,33 +20,38 @@ recurrence period.
 Create recurrence periods as product variants
 =============================================
 
-To set up each recurrence period as a product variant, go to :menuselection:`Subscriptions -->
-Subscriptions --> Products` and select a product. In the :guilabel:`Attributes & Variants` tab,
-click :guilabel:`Add a line`.
+To set up each recurrence period as a product variant, go to :menuselection:`Subscriptions app -->
+Products`, and select a product. In the :guilabel:`Attributes & Variants` tab, click :guilabel:`Add
+a line`.
 
-Create an :guilabel:`Attribute` called `Billing Period` (or something similar) by typing in the
-name and clicking :guilabel:`Create`. This attribute name appears as the option heading on the
-product page of the eCommerce shop.
+Then proceed to create an :guilabel:`Attribute` called `Billing Period` (or something similar), by
+typing in the name, and clicking :guilabel:`Create` from the mini drop-down menu that appears. This
+attribute name appears as option heading on the product page of the eCommerce shop.
```

Comment:
Totally personal preference, but when it comes to naming things, since users don't HAVE to call it `Billing Period`, let's try describing what they should type in, instead of telling them what to type.
```suggestion
Then proceed to create an :guilabel:`Attribute` by typing in a name to represent the billing period, and clicking :guilabel:`Create` from the mini drop-down menu that appears. This
attribute name appears as option heading on the product page of the eCommerce shop.
```

Reply:
I totally understand (and appreciate) where you're coming from. I, too, feel that same way. Personally, I think my parenthetical suggestion of "(or something similar)" suffices in this regard, and doesn't make the user feel beholden to one *specific* title. So, I think I'm going to leave it as-is, but thanks for the suggestion. 👍